src/theme/app/THEME.md

```diff
@@ -0,0 +1,65 @@
+## Adding dark theme for components
+
+1. To differentiate between the colors of dark and light theme, we have created a customization system in which we can define styles like color, position, display etc using {props => props.theme.DarkTheme? darkThemeStyle: LightThemeStyle} 
+  
+
+2. It is recommended to use pre-existing styles or colors from the theme. Avoid adding your own CSS variables."
+
+
+ Examples- To change Text Color according to theme
+
+color: ${props => props.theme.DarkTheme ? "rgba(255, 255, 255, 0.6)" :"rgba(0, 0, 0, 0.6)"}; (Not recommended, only use when there is a need for creating a new css variable)
+
+color: ${props => props.theme.DarkTheme ? props.theme.secondaryColor : props.theme.primaryLightColor}; (Recommended)
+
+
+
+## Changing svg according to theme
+
+1. To Change svg according to theme, We strongly recommend innerpaths or rect or polygon to fill them with the preferable colors like we did in the case of the components.
```

Comment:
Can you explain the approach you have adopted to display different svg's in different modes first. It can be very brief but will provide a better understanding to contributors. After explaining what you did and why, we can move on to this instruction/example.